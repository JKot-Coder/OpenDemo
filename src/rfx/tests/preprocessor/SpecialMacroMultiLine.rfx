// :RUN_RFXC -P %STD_OUTPUT% %INPUT% 

#define M(ARG) __LINE__

int x = M
(
ignored
)
;