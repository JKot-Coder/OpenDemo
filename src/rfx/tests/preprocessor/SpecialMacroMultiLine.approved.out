SourceOutput:[
    #line 5 "../src/rfx/tests/preprocessor/SpecialMacroMultiLine.rfx"
    int x = 5
    
    
    
    ;
]
Diagnostic:[
]
